src/Microsoft.TemplateEngine.Core.Contracts/IGlobalRunSpec.cs

```diff
@@ -18,7 +18,7 @@ public interface IGlobalRunSpec
 
         IReadOnlyDictionary<string, IReadOnlyList<IOperationProvider>> LocalizationOperations { get; }
 
-        string PlaceholderFilename { get; }
+        IReadOnlyList<string> IgnoreFileNames { get; }
```

Comment:
This could potentially be a hashset, since it appears its main use is via Contains(). But in general, it will probably only have a few entries max, so a list is probably as performant.